app/api/v1/revalidate/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { revalidateTag } from 'next/cache';
+
+export async function GET(request: NextRequest) {
+	const tag = request.nextUrl.searchParams.get('tag');
+
+	if (!tag) {
+		return NextResponse.json({
+			revalidated: false,
+			msg: 'you need to provide query param ?tag={cache-tag}',
+		});
+	}
+	revalidateTag(tag);
+	console.log(`cache invalidated for tag ${tag} at ${Date.now()}`);
```

Comment:
We should use a logger here for monitoring tools (e.g. datadog) to capture invalidating events for easier investigation.